### PR TITLE
[Accton]: Add a new supported device AS5712-54X

### PR DESCRIPTION
**Pull request by jeankuo**
Switch Vendor: Edge-core
Switch SKU: AS5712-54X
ASIC Vendor: Broadcom
Swich ASIC: TRIDENT2
Port Configuration: 48x10G+6x40G
SONiC Image: SONiC-ONIE-Broadcom

### CODE REVIEW

File: device/accton/x86_64-accton_as5712_54x-r0/led_proc_init.soc [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+ 57 32 0F 87 32 0F 87 32 0F 87 32 0F 87 57 00 00
+led 1 start
+
+# Init procedure could have some unexpective behavior for portd state.
+# So we need to disable all ports and then enable all ports to reset the ports s
+tate.
+port xe enable=no
+port xe enable=yes
+
+m xlmac_tx_ctrl average_ipg=0xc
```
Comment (lguohan) [inline device/accton/x86_64-accton_as5712_54x-r0/led_proc_init.soc]: not sure why you need this.

File: device/accton/x86_64-accton_as5712_54x-r0/led_proc_init.soc [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+# Set default interface mode for 10G SFP ports
+port xe sp=10000 if=SFI
+console off; port xe sp=40000 if=CR4; console on
```
Comment (lguohan) [inline device/accton/x86_64-accton_as5712_54x-r0/led_proc_init.soc]: is the link speed 40g or 10g?

File: device/accton/x86_64-accton_as5712_54x-r0/led_proc_init.soc [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+ 85 28 67 56 57 32 0F 87 57 12 A0 F8 15 1A 01 71 \
+ A1 28 67 56 80 28 67 56 80 28 67 56 80 28 67 56 \
+ 57 32 0F 87 32 0F 87 32 0F 87 32 0F 87 57 00 00
+led 1 start
+
+# Init procedure could have some unexpective behavior for portd state.
+# So we need to disable all ports and then enable all ports to reset the ports s
+tate.
+port xe enable=no
+port xe enable=yes
```
Comment (lguohan) [inline device/accton/x86_64-accton_as5712_54x-r0/led_proc_init.soc]: what is the init problem? sai issue, opennsl issue?

Comment (jeankuo) [inline device/accton/x86_64-accton_as5712_54x-r0/led_proc_init.soc]: OK. I will remvoe it.

File: device/accton/x86_64-accton_as5712_54x-r0/led_proc_init.soc (hunk 2 above)
Comment (jeankuo) [inline device/accton/x86_64-accton_as5712_54x-r0/led_proc_init.soc]: Is 10G. I will remvoe it.

File: device/accton/x86_64-accton_as5712_54x-r0/led_proc_init.soc (hunk 1 above)
Comment (jeankuo) [inline device/accton/x86_64-accton_as5712_54x-r0/led_proc_init.soc]: OK. I will remvoe it.

File: device/accton/x86_64-accton_as5712_54x-r0/led_proc_init.soc (hunk 3 above)
Comment (jeankuo) [inline device/accton/x86_64-accton_as5712_54x-r0/led_proc_init.soc]: Done to update.

File: device/accton/x86_64-accton_as5712_54x-r0/Accton-AS5712-54X/port_config.ini [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+Ethernet62     107                tenGigE62
+Ethernet63     108                tenGigE63
+Ethernet64     109                tenGigE64
+Ethernet65     110                tenGigE65
+Ethernet66     111                tenGigE66
+Ethernet67     112                tenGigE67
+Ethernet68     77                 tenGigE68
+Ethernet69     78                 tenGigE69
+Ethernet70     79                 tenGigE70
+Ethernet71     80                 tenGigE71
```
Comment (jleveque) [inline device/accton/x86_64-accton_as5712_54x-r0/Accton-AS5712-54X/port_config.ini]: Please add newline at EOF.

Comment (jeankuo) [inline device/accton/x86_64-accton_as5712_54x-r0/Accton-AS5712-54X/port_config.ini]: done to change.

**Comment by pollyhsu2git** [comment]
@stcheng, we just found that our *config.bcm* has not been packed within our latest libsaibcm_3.0.3.2-5_amd64.deb (checked your latest jenkins#360) 
sonic-v2-as5712-git j_360-libsaibcm_3 0 3 2-5-pr898

**Pull request merged**